boefjes/boefjes/plugins/kat_leakix/normalize.py

```diff
@@ -44,10 +44,15 @@ def run(normalizer_meta: NormalizerMeta, raw: bytes | str) -> Iterable_[OOI]:
         event_ooi = pk_ooi
 
         # Autonomous System
-        as_number = event["network"]["asn"]
+        as_number = event["network"].get("asn", False)
```

Comment:
`as_number` isn't a boolean, so if it doesn't have a value it should be set to `None` instead of `False` which is what `get` returns if you don't specify a default value.

```suggestion
        as_number = event["network"].get("asn")
```